user_tools/src/spark_rapids_pytools/resources/azure-instances-catalog.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
- Are those VM instances depend on the region? Or they are available on all regions/zones?
- It can be useful to add a comment at the beginning of the file describing how the file is generated. 
   - If there is a script that generates that json file, then we need this to be a part of the repo. Perhaps, a "`dev/scripts`" folder should include those scripts.
   - The reason we need that script is the possibility of extending in the future. For example, is this ia subset of the VMs, or this is all available Azure VMs.
- I understand that these entries are not specific to Databricks. then the question is Does Databricks accept all Azure VMs? If not, then there should be another map in azure-databricks configs listing the VM-keys that are available on databricks.

Reply:
- They depend on regions.
- I will follow up with this point.
- We can find and only process the VMs supported on Databricks.